views/layouts/partials/_header.php

```diff
@@ -111,11 +111,12 @@ class="fa fa-inverse fa-bars"></i></button>
                         ['label' => 'Learn', 'options' => ['class' => 'separator']],
                         ['label' => 'Books', 'url' => ['site/books']],
                         ['label' => 'Resources', 'url' => ['site/resources']],
-                        ['label' => 'Develop', 'options' => ['class' => 'separator']],
+                        ['label' => 'Development', 'options' => ['class' => 'separator']],
```

Comment:
```suggestion
                        ['label' => 'Develop', 'options' => ['class' => 'separator']],
```

Reply:
I thought that after adding "Donate" subitem this would be more appropriate name for parent section. But If you are fine with it, I'm fine too.